src/components/DappBrowser/ErrorPage.tsx

```diff
@@ -0,0 +1,52 @@
+import lang from 'i18n-js';
+import React from 'react';
+import { Centered } from '../layout';
+import { Text } from '../text';
+import styled from '@/styled-thing';
+import { Colors } from '../../styles/colors';
+import { Box } from '@/design-system';
+import { useTheme } from '@/theme';
+import { View } from 'moti';
+
+const BodyText = styled(Text).attrs(({ theme: { colors } }: { theme: { colors: Colors } }) => ({
+  align: 'center',
+  color: colors.alpha(colors.blueGreyDark, 0.6),
+  lineHeight: 'loosest',
+  size: 'medium',
+}))({
+  paddingBottom: 23,
+  paddingTop: 4,
+});
```

Comment:
Possible to use the `Text` import from `@/design-system` instead here?

Reply:
This is just a placeholder. Will do once we get designs